examples/provider/bid/audience-network/src/audience-network-bid-delegate.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * @file Audience Network BidDelegate.
+ */
+import 'whatwg-fetch';
+
+const name = 'audience-network';
+
+/**
+ * Does this slot contain valid parameters?
+ * @param {Object} slot
+ * @returns {Boolean}
+ */
+const validateSlot = slot =>
+  typeof slot === 'object' &&
+  typeof slot.name === 'string' &&
+  typeof slot.elementId === 'string' &&
+  Array.isArray(slot.sizes) &&
+  slot.sizes.length > 0 &&
+  typeof slot.getParam === 'function' &&
+  typeof slot.getParam(name) === 'object' &&
+  typeof slot.getParam(name).placementId === 'string';
+
+/**
+ * Return a copy of a bid with slot sizes flattened and filtered
+ * @param {Object} bid
+ * @returns {Object} copy of bid
+ */
+const flattenBidRequestSizes = bid => {
+  const sizes = Array.isArray(bid.sizes) && bid.sizes
+    .map(flattenSize)
+    .filter(isValidSize);
+  return Object.assign({}, bid, { sizes });
```

Comment:
`Object.assign` for IE is primarily an Edge browser capability IE10/9 will have an issue with this.

_See also_ comment for function `validateSlot`